docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -1272,199 +903,104 @@ All commands should be executed on a master node of the existing cluster.
-
-   * Retrieve the value of `SE_REGISTRY_PACKAGE_PROXY`:
-
-     ```shell
-     SE_REGISTRY_PACKAGE_PROXY=$(kubectl exec se-image -- cat deckhouse/candi/images_digests.json | jq -r ".registryPackagesProxy.registryPackagesProxy")
-     ```
-
-     Run the command:
-
-     ```shell
-     sudo /opt/deckhouse/bin/crictl pull registry.deckhouse.io/deckhouse/se@$SE_REGISTRY_PACKAGE_PROXY
-     ```
-
-     Example output:
-
-     ```console
-     Image is up to date for sha256:7e9908d47580ed8a9de481f579299ccb7040d5c7fade4689cb1bff1be74a95de
-     ```
-
-   * Retrieve the value of `SE_MODULES`:
-
-     ```shell
-     SE_MODULES=$(kubectl exec se-image -- ls -l deckhouse/modules/ | grep -oE "\d.*-\w*" | awk {'print $9'} | cut -c5-)
-     ```
-
-     Check the result of the command to make sure it was successful:
-
-     ```shell
-     echo $SE_MODULES
-     ```
-
-     Example output:
-
-     ```console
-     common priority-class deckhouse external-module-manager ...
-     ```
-
-   * Retrieve the value of `USED_MODULES`:
-
-     ```shell
-     USED_MODULES=$(kubectl get modules -o custom-columns=NAME:.metadata.name,SOURCE:.properties.source,STATE:.properties.state,ENABLED:.status.phase | grep Embedded | grep -E 'Enabled|Ready' | awk {'print $1'})
-     ```
-
-     Check the result of the command to make sure it was successful:
-
-     ```shell
-     echo $USED_MODULES
-     ```
-
-     Example output:
-
-     ```console
-     admission-policy-engine cert-manager chrony ...
-     ```
-
-   * Form the list of modules that must be disabled:
-
-     ```shell
-     MODULES_WILL_DISABLE=$(echo $USED_MODULES | tr ' ' '\n' | grep -Fxv -f <(echo $SE_MODULES | tr ' ' '\n'))
-     ```
-
-1. Verify that the modules in use are supported by the SE edition.
-   Run the following command to display modules that are not supported by SE and will be disabled:
-
-   ```shell
-   echo $MODULES_WILL_DISABLE
+   kubectl run $NEW_EDITION-image --image=registry.deckhouse.io/deckhouse/$NEW_EDITION/install:$DECKHOUSE_VERSION --command sleep --infinity
    ```
 
-   > Check the list and make sure the functionality of these modules is not required in your cluster or that you are prepared to disable them.
-
-   Disable the unsupported modules:
+1. Once the pod is in `Running` state, execute the following commands:
 
    ```shell
-   echo $MODULES_WILL_DISABLE | 
-     tr ' ' '\n' | awk {'print "kubectl -n d8-system exec deploy/deckhouse -- deckhouse-controller module disable",$1'} | bash
+   NEW_EDITION_MODULES=$(kubectl exec $NEW_EDITION-image -- ls -l deckhouse/modules/ | grep -oE "\d.*-\w*" | awk {'print $9'} | cut -c5-)
+   USED_MODULES=$(kubectl get modules -o custom-columns=NAME:.metadata.name,SOURCE:.properties.source,STATE:.properties.state,ENABLED:.status.phase | grep Embedded | grep -E 'Enabled|Ready' | awk {'print $1'})
+   HANDLER=$(kubectl exec $NEW_EDITION-image -- cat deckhouse/candi/images_digests.json | jq -r ".deckhouse.webhookHandler")
```

Comment:
As I can see you dont use HANDLER in following steps. Maybe we dont need to set this variable? And it is for DECKHOUSE_KUBE_RBAC_PROXY, DECKHOUSE_INIT_CONTAINER the same?